docs/doc/12-load-data/00-stage.md

```diff
@@ -1,201 +1,209 @@
-| name         | stage_type | stage_params                                                                                                                                                | copy_options                                  | file_format_options                                                                                                | comment |
-+--------------+------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------+---------+
-| my_int_stage | Internal   | StageParams { storage: S3(StageS3Storage { bucket: "", path: "", credentials_aws_key_id: "", credentials_aws_secret_key: "", encryption_master_key: "" }) } | CopyOptions { on_error: None, size_limit: 0 } | FileFormatOptions { format: Parquet, skip_header: 0, field_delimiter: ",", record_delimiter: "\n", compression: None } |         |
-+--------------+------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------+--------------------------------------------------------------------------------------------------------------------+---------+
-```
+Before you start, make sure you have completed the following tasks:
 
-### Step 2. Stage the Data Files
+- Download and save the sample file [books.parquet](https://datafuse-1253727613.cos.ap-hongkong.myqcloud.com/data/books.parquet) to a local folder. The file contains two records:
 
-Download the sample data file(Choose CSV or Parquet), the file contains two records:
 ```text
 Transaction Processing,Jim Gray,1992
 Readings in Database Systems,Michael Stonebraker,2004
 ```
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+- Create a table with the following SQL statements in Databend:
 
-<Tabs groupId="sample-data">
+```sql
+USE default;
+CREATE TABLE books
+(
+    title VARCHAR,
+    author VARCHAR,
+    date VARCHAR
+);
+```
 
-<TabItem value="csv" label="CSV">
+## Tutorial 1: Loading from User Stage
 
-Download [books.csv](https://datafuse-1253727613.cos.ap-hongkong.myqcloud.com/data/books.csv)
+Follow this tutorial to upload the sample file to the user stage and load data from the staged file into Databend.
 
-</TabItem>
+### Step 1: Upload Sample File
 
-<TabItem value="parquet" label="Parquet">
+1. Use cURL to make a request to the [File Upload API](../11-integrations/00-api/10-put-to-stage.md):
```

Comment:
It would be better to add we can upload files to stage with some Databend driver.

Reply:
bendsql?

Reply:
We could do it in the another PR, also we should add a command 'PUT STAGE' to https://databend.rs/doc/sql-commands/ddl/stage/ with bendsql or others driver.